app-misc/vcontrold/vcontrold-0.98.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+man +vclient vsim"
+
+RDEPEND="dev-libs/libxml2"
+DEPEND="${RDEPEND}
+	man? ( dev-python/docutils )"
+
+src_prepare() {
+	sed "s/@VERSION@/${PV}/" "src/version.h.in" \
+		> "src/version.h" || die "Setting version failed"
```

Comment:
Okay, I see why the redirect is needed, now.

You can drop the quotes around the path names since you don't have to worry about spaces screwing things up.